### PR TITLE
[HZ-1064] Hazelcast configs to specify the CP member priority

### PR DESCRIPTION
**Pull request by arodionov**
Configs update for the new CP member property `cpMemberPriority`

Checklist:
- [x] Labels (`Team:`, `Type:`, `Source:`, `Module:`) and Milestone set
- [x] Label `Add to Release Notes` or `Not Release Notes content` set
- [x] Request reviewers if possible

### CODE REVIEW

File: hazelcast/src/test/resources/com/hazelcast/config/yaml/testcases/member/cpsubsystem/range-failures.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
         "session-heartbeat-interval-seconds": 0,
         "missing-cp-member-auto-removal-seconds": -1,
         "data-load-timeout-seconds": 0,
+        "cp-member-priority": 0,
```
Comment (ramizdundar) [inline hazelcast/src/test/resources/com/hazelcast/config/yaml/testcases/member/cpsubsystem/range-failures.json:11]: minor: I think we don't need this as it's not causing range failure.

**Review by ramizdundar — APPROVED** [review]
This is the PR I will take a look again for my future static config changes :D, LGTM.

File: hazelcast-spring-tests/src/test/resources/com/hazelcast/spring/fullConfig-applicationContext-hazelcast.xml [hunk 2]
```diff
@@ -1029,6 +1029,7 @@
                 <hz:persistence-enabled>false</hz:persistence-enabled>
                 <hz:base-dir>/custom-dir</hz:base-dir>
                 <hz:data-load-timeout-seconds>30</hz:data-load-timeout-seconds>
+                <hz:cp-member-priority>-1</hz:cp-member-priority>
```
Comment (frant-hartm) [inline hazelcast-spring-tests/src/test/resources/com/hazelcast/spring/fullConfig-applicationContext-hazelcast.xml:1032]: Are negative values allowed intentionally?

**Review by frant-hartm — APPROVED** [review]
LGTM, just one question.

File: hazelcast-spring-tests/src/test/resources/com/hazelcast/spring/fullConfig-applicationContext-hazelcast.xml (hunk 2 above)
Comment (arodionov) [reply hazelcast-spring-tests/src/test/resources/com/hazelcast/spring/fullConfig-applicationContext-hazelcast.xml:1032]: In theory yes, the default value for priorities is 0. If someone decides to make one CP member a non-priority, the easiest way would be to set a negative priority (eg -1).

Comment (arodionov) [reply hazelcast-spring-tests/src/test/resources/com/hazelcast/spring/fullConfig-applicationContext-hazelcast.xml:1032]: Thanks for the review!

**Review by ramizdundar — APPROVED** [review]
I don't know why but this PR seen as to be reviewed in my PR list, hence I'm approving for the 2nd time.

**Pull request merged**